databases/week3/homework_week3.sql

```diff
@@ -0,0 +1,134 @@
+CREATE database meal_sharing;
+use meal_sharing;
+CREATE TABLE meal(
+	`id` INT UNSIGNED AUTO_INCREMENT,
+    `title` VARCHAR(255) NOT NULL,
+    `description` TEXT(255) NOT NULL,
+    `location` VARCHAR(255) NOT NULL,
+    `when` DATETIME NOT NULL,
+    `max_reservations` INT UNSIGNED NOT NULL,
+    `price` DECIMAL(10,2) NOT NULL,
+    `created_date` DATE NOT NULL,
+    PRIMARY KEY (id)
+) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+
+CREATE TABLE reservation (
+    `id` INT UNSIGNED AUTO_INCREMENT,
+    `number_of_guests` INT UNSIGNED NOT NULL,
+    `meal_id` INT UNSIGNED NOT NULL,
+    `created_date` DATE NOT NULL,
+    `contact_phonenumber` VARCHAR(255) NOT NULL,
+    `contact_name` VARCHAR(255) NOT NULL,
+    `contact_email` VARCHAR(255) NOT NULL,
+    PRIMARY KEY (id),
+    CONSTRAINT `fk_reservation_meal` FOREIGN KEY (`meal_id`) REFERENCES `Meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+    ) ENGINE = InnoDB DEFAULT CHARSET = utf8mb4 COLLATE = utf8mb4_unicode_ci;
+    
+    CREATE TABLE review (
+    `id` INT UNSIGNED AUTO_INCREMENT,
+    `title` VARCHAR(255) NOT NULL,
+    `description` TEXT(255) NOT NULL,
+    `meal_id` INT UNSIGNED NOT NULL,
+    `stars` INT UNSIGNED NOT NULL,
+    `created_date` DATE NOT NULL,
```

Comment:
Just to avoid ambiguity,  it would be nice if you can name the attribute `created_date` differently, like `created` because there is a similar attribute name under the meal table.